fontforgeexe/charinfo.c

```diff
@@ -3251,8 +3251,11 @@ return( NULL );
 
     if ( ymax<=ICON_WIDTH ) ymax = ICON_WIDTH;
     if ( ymin>0 ) ymin = 0;
-    if ( xmax<xmin )
+    if ( xmax<xmin ) {
+        free(extras);
```

Comment:
I saw that, near the end of the function, we free the contents of extra before freeing extra. Is there a reason to skip that here?